lms/templates/student_account/register.underscore

```diff
@@ -1,12 +1,16 @@
 <div class="js-form-feedback" aria-live="assertive" tabindex="-1">
 </div>
 
-<div class="toggle-form">
-    <span class="text"><%- edx.StringUtils.interpolate(gettext('Already have an {platformName} account?'), {platformName: context.platformName }) %></span>
-    <a href="#login" class="form-toggle" data-type="login"><%- gettext("Sign in.") %></a>
-</div>
+<% if (!context.syncLearnerProfileData) { %>
+	<div class="toggle-form">
+		<span class="text"><%- edx.StringUtils.interpolate(gettext('Already have an {platformName} account?'), {platformName: context.platformName }) %></span>
+		<a href="#login" class="form-toggle" data-type="login"><%- gettext("Sign in.") %></a>
+	</div>
+<% } %>
+
+<h2><%- gettext('Create an Account')%></h2>
 
-<form id="register" class="register-form" autocomplete="off" tabindex="-1" method="POST">
+<form id="register" class="register-form <% if (context.syncLearnerProfileData) { %> sync-profile-registration-form <% } %>" autocomplete="off" tabindex="-1" method="POST">
```

Comment:
Why do you need to add this class?

Reply:
We need it to apply css styles to disabled input fields, labels etc.

Reply:
But why do the styles for disabled input fields, labels, etc. need to depend on a class related to syncing profile information? A disabled field should be a disabled field regardless of why it is disabled.